search/ranking_utils.hpp

```diff
@@ -92,4 +148,30 @@ NameScore GetNameScore(std::vector<strings::UniString> const & tokens, Slice con
 }
 
 string DebugPrint(NameScore score);
+
+// Returns maximum number of errors that were made during matching
```

Comment:
What do you mean by "maximum" here?

Reply:
"maximum" is by mistake here. The correct word is "total".